e2e_deployment_files/nestededge_bottomLayerBaseDeployment_connectivity_mqtt.template.json

```diff
@@ -645,6 +645,27 @@
               "image": "$upstream:443/microsoft/azureiotedge-metrics-collector:<Build.BuildNumber>-linux-<Architecture>",
               "createOptions": "{\"HostConfig\":{\"NetworkMode\":\"host\"},\"NetworkingConfig\":{\"EndpointsConfig\":{\"host\":{}}}}"
             }
+          },
+          "genericMqttTester": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "TEST_SCENARIO": {
+                "value": "InitiateAndReceiveRelayed"
+              },
+              "TRACKING_ID": {
+                "value": "<TrackingId>"
+              },
+              "TEST_START_DELAY": {
+                "value": "120s"
+              }
```

Comment:
This test start delay is not passed through the pipeline because of differences in rust module Duration formatting. There is a work item to make this consistent with other test module start delays.

But right now it causes no problems.